csv2rdf/acousticbrainz/ab_reconciliation_details.md

```diff
@@ -0,0 +1,9 @@
+#   Properties:
+The reconciliation of AcousticBrainz properties are not specific. 
```

Comment:
What does this sentence mean?

Reply:
There are no properties like genre_electronic, genre_rosameria, etc. So I can only use P136 for all the genres.